rsc/extra/travis_js_test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+set -eu
+
+# === check_javascript ===
+#
+# This script is intendend to check that alt-ergo-js builds
+# and run correctly with nodejs
+
+# Be sure of where we are, and initialize some variables
+git_repo=`git rev-parse --show-toplevel`
+local_install_dir=$1
+
+# Cd into the root project directory
+cd $git_repo
+
+# Configure using the local installation directory
+./configure --prefix=$local_install_dir
+
+# Install requiered package for js compilation
+opam install zarith_stubs_js js_of_ocaml-compiler js_of_ocaml lwt js_of_ocaml-lwt js_of_ocaml-ppx lwt_ppx
```

Comment:
It would be better to pin (with no action) the opam package (which probably needs to be created) for the js version of alt-ergo, and then install its dependencies, that way the script also checks that the specified dependencies are enough to compile it.

Reply:
I've add 3 packages for js, when the tests will pass i will merge in one as you suggested

Reply:
Done